src/lib/OpenEXRCore/internal_b44.c

```diff
@@ -489,6 +489,14 @@ compress_b44_impl (exr_encode_pipeline_t* encode, int flat_field)
         scratch += nBytes;
     }
 
+    if (nOut > encode->packed_bytes)
+    {
+        memcpy (
```

Comment:
I think I didn't encounter this because I don't support righting b44 files.... !!

Reply:
yeah, i only noticed thanks to having a test that had a height % 4 == 1 (i.e. single scanline vs compressing the 4x4 block of 0s)